src/main.tsx

```diff
@@ -17,6 +17,10 @@ const ROUTER = createBrowserRouter([
     path: "/login",
     element: <LoginPage />,
   },
+  {
+    path: "/signup",
+    element: <Signup />,
```

Comment:
Follow the naming convention for page
`SignupPage`

pages should come from the pages directory and not from the features
check `preprod` for sample